documentation/http-behaviors.md

```diff
@@ -7,12 +7,12 @@ uid: htpBehaviors
 While the OMF specification does not define a particular binary message protocol, OSIsoft systems accepting OMF over HTTP conform to a set of behaviors described in this section.
 ## Message Size
 
-The body of an OMF message has a maximum size of 192KB. A request with a body size exceeding the maximum will be rejected with an HTTP response code 
+The body of an OMF message has a maximum size of 192KB. A request with a body size exceeding the maximum will be rejected with an HTTP response code
```

Comment:
instead of removing space shouldn't we move it to the same line? "HTTP response code of `413 Payload Too Large`."

Reply:
I'm not sure what line length is being targeted, so I figured it was safest to leave it. I have removed the newline and re-inserted the space, so now this section is all on one line.